documentation/architecture/achecitecture-decision-records/data-storage.md

```diff
@@ -0,0 +1,33 @@
+# Data Storage
+
+## Context and Problem Statement
+
+Perhaps the most important decision of any service is how the data will be stored. The factors to be considered include:
+
+* Storage costs
+* Bandwidth costs
+* Suitability to the service usage
+* Ease of data ingestion 
+
+## Considered Options
+- Azure Cosmos DB, a fully-managed and distributed NoSQL database
+- Azure Managed SQL, a fully-managed and serverless distributed SQL database
+- Azure Blob Storage, an object storage solution
+- Azure Synapse, a Data Warehousing solution
+
+### Evaluation
+
+| Criteria | Comment | Cosmos DB | SQL | Blob Storage | Synapse | 
```

Comment:
the evaluation table doesn't seems to align correctly on my end. attaching image for  reference. 
<img width="497" alt="image" src="https://github.com/DFE-Digital/education-benchmarking-and-insights/assets/111439003/407b51e7-7576-4de2-94d2-dd7e8830d357">

Reply:
There is space missing after after total column. Image for reference attached. 
<img width="522" alt="image" src="https://github.com/DFE-Digital/education-benchmarking-and-insights/assets/111439003/d842c392-fce9-40cf-89b0-87a952edb78f">

Reply:
Not sure why you're seeing that, as I see this:
<img width="520" alt="Screenshot 2024-01-12 095503" src="https://github.com/DFE-Digital/education-benchmarking-and-insights/assets/98596456/259d137c-2ee0-4ac7-805d-ffd51f48de9d">

Reply:
thats strange. I could still see the old way and adding a space to it fixes it for me. if adding a space doesn't break it for you then would be good to add it.

Reply:
@faizi I have added the spaces